src/common/storage/backends/sciserver-files/metadata.json

```diff
@@ -26,6 +26,16 @@
             "value": "USERNAME/deepforge_data",
             "valueType": "string",
             "readOnly": false
+        },
+        {
```

Comment:
Hmm... It isn't clear to me that it is better to add another option for users... I think it might be better to just support a sort of absolute and relative volume specification. Since there are only two different volume pools, we could simply check if the volume starts with `/?(Temporary|Storage)`. If not, add `/Storage/<username>` to the beginning of the volume and use that. This would let us change the default value for the previous option to `deepforge_data` which would be nice as we wouldn't need to edit `USERNAME` each time (the issue you opened before). :)

That said, it would be nice to make this clear in the description for the previous option. Maybe something like "Volume to use for upload (eg, deepforge_data, /Temporary/someUser/deepforge_tmp)"

Reply:
Ok. But a clearer option would be a config value IMO. Especially, since you can create volume of same name under different pools example `Temporary/A` or `Storage/A`. But that could be achieved using what you suggested as well and not necessary to create a different config.

Reply:
Yeah, it isn't really clear about the best approach here as there are pros and cons to each approach: 
Using an additional option:
- explicit and easy for people to use temporary storage if they choose
- differences are listed explicitly in the description
- users will always need to edit "USERNAME" since we cannot determine if the volume contains the username.

Inferring it from the file name:
- Default value for the volume will work out of the box
- volume info can basically be copy-pasted from the SciServer Files page:
![DeepinScreenshot_select-area_20200410165825](https://user-images.githubusercontent.com/4982789/79025854-9376f880-7b4c-11ea-886f-0835774770da.png)

That said, addressing #1638 might make editing "USERNAME" less of an issue...

Reply:
Yeah, I think listing the differences is a nice feature of this approach and I think that the annoyance of repetitive filling out prompts can be addressed uniformly for all the dialogs as in #1648